oidc-service-impl/src/test/java/io/mosip/esignet/services/ConsentHelperServiceTest.java

```diff
@@ -482,6 +481,85 @@ public void processConsent_withEmptyRequestedClaims_thenPass(){
         Assert.assertEquals(oidcTransaction.getConsentAction(),ConsentAction.NOCAPTURE);
     }
 
+    @Test
+    public void getConsentAction_withValidConsentAndGetConsentActionAsNoCapture_thenPass() throws Exception {
```

Comment:
We might not need these test cases, if the method is not duplicated.

Reply:
removed